components/event/content.php

```diff
@@ -45,12 +48,16 @@
 				</div>
 				<?php } ?>
 
+				<?php if ( ! empty( $event_venue['link'] ) ) { ?>
 				<span class="card-directions">
-					<a href="#">View directions</a>
+					<a href="<?php echo esc_url( $event_venue['link'] ); ?>">View directions</a>
```

Comment:
"View directions" could probably be something else here. Maybe "View location in Google Maps"?

Reply:
I like it!